internal/support/scheduler/operators/interval/add.go

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright 2019 VMware Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+package interval
+
+import (
```

Comment:
Please sort these imports.

Reply:
Aren't they sorted? Do named imports need to be on top?

Reply:
This is fine. We don't sort by the alias but rather by the import path.